.code-samples.meilisearch.yaml

```diff
@@ -660,22 +660,24 @@ geosearch_guide_sort_usage_2: |-
       "rating:desc",
     },
   })
-document_guide_create_index_primary_key: |-
+primary_field_guide_create_index_primary_key: |-
   client.CreateIndex(&meilisearch.IndexConfig{
     Uid: "movies",
     PrimaryKey: "reference_number",
   })
-document_guide_add_document_primary_key: |-
+primary_field_guide_add_document_primary_key: |-
   documents := []map[string]interface{}{
     {
       "reference_number": 287947,
-      "title":            "Shazam",
-      "poster":           "https://image.tmdb.org/t/p/w1280/xnopI5Xtky18MPhK40cZAGAOVeV.jpg",
-      "overview":         "A boy is given the ability to become an adult superhero in times of need with a single magic word.",
-      "release_date":     "2019-03-23",
+      "title":            "Diary of a Wimpy Kid",
+      "author":           "Jeff Kinney",
+      "genres":           []string{"comedy", "humor"},
+      "price":     5.00,
```

Comment:
I tried to indent it correctly but it is very hard with github. Do you want to align it with the above values?